### PR TITLE
Bibliography formatting fixes

### PR DESCRIPTION
**Pull request by mhoemmen**
Fix spelling and formatting in bibliography.

These fixes were separated from PR #7504 per [reviewer request](https://github.com/cplusplus/draft/pull/7504#discussion_r1892801089).

### CODE REVIEW

**Comment by tkoeppe** [comment]
All done with this?

**Review by jensmaurer — CHANGES_REQUESTED** [review]
Please rephrase the commit descriptions: https://github.com/cplusplus/draft/wiki/Commit-message-format

File: source/back.tex [hunk 1]
```diff
@@ -52,11 +52,11 @@
...
 \bibitem{kr}
-  Brian W.\ Kernighan and Dennis M. Ritchie,
+  Brian W.\ Kernighan and Dennis M.\ Ritchie,
   \doccite{The C Programming Language}, Appendix A\@.
   Prentice-Hall, 1978, ISBN 0-13-110163-3, copyright \copyright 1978 AT\&T
 \bibitem{cpp-lib}
-  P.J.\ Plauger,
+  P.\,J.\ Plauger,
   \doccite{The Draft Standard \Cpp{} Library}.
   Prentice-Hall, ISBN 0-13-117003-1, copyright \copyright 1995 P.J.\ Plauger
```
Comment (tkoeppe) [inline source/back.tex:61]: If you're going to change L59, then also here?

**Comment by tkoeppe** [comment]
> Please rephrase the commit descriptions: https://github.com/cplusplus/draft/wiki/Commit-message-format

Don't worry, I'll do that when squashing here.

**Comment by tkoeppe** [comment]
Once it's in, you can rebase and squash the other PR.

File: source/back.tex (hunk 1 above)
Comment (mhoemmen) [reply source/back.tex]: Done; thanks! : - )

**Comment by mhoemmen** [comment]
@tkoeppe wrote:
> All done with this?

All done; thanks for asking!

@jensmaurer wrote:
> Please rephrase the commit descriptions....

Done!

**Review by jensmaurer — APPROVED** [review]

**Pull request merged**